apps/ledger-live-mobile/e2e/models/wallet/portfolioPage.ts

```diff
@@ -43,4 +44,8 @@ export default class PortfolioPage {
   async openViaDeeplink() {
     await openDeeplink(baseLink);
   }
+
+  openMarketPage() {
```

Comment:
Given that this tab bar component is visible not just in the portfolio page I think you should have the action in the `walletTabNavigator.ts` below

Reply:
Ah, I see the behaviour now if you enable/disable the feature flag. Fine to leave as is 👍